src/Client/BuzzHouse/Generator/QueryOracle.cpp

```diff
@@ -493,28 +310,30 @@ int QueryOracle::generateFirstSetting(RandomGenerator & rg, SQLQuery & sq1)
     nsettings.clear();
     for (uint32_t i = 0; i < nsets; i++)
     {
-        const TestSetting & ts = rg.pickRandomlyFromVector(test_settings);
+        const std::string & setting = rg.pickKeyRandomlyFromMap(queryOracleSettings);
+        const CHSetting & chs = queryOracleSettings.at(setting);
         SetValue * setv = i == 0 ? sv->mutable_set_value() : sv->add_other_values();
 
-        setv->set_property(ts.tsetting);
-        if (ts.options.size() == 2)
+        setv->set_property(setting);
```

Comment:
Another round of non-conformant method name. Should be `setProperty`

Reply:
it is a method generated by Protobuf